lib/node/runner/jsonrpc.go

```diff
@@ -0,0 +1,177 @@
+package runner
+
+import (
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+	"github.com/gorilla/rpc"
+	jsonrpc "github.com/gorilla/rpc/json"
+
+	"boscoin.io/sebak/lib/common"
+	"boscoin.io/sebak/lib/storage"
+)
+
+const MaxLimitListOptions uint64 = 10000
+
+type EchoArgs string
+type EchoResult string
+
+type DBHasArgs string
+type DBHasResult bool
+
+type DBGetArgs string
+type DBGetResult storage.IterItem
+
+type GetIteratorOptions struct {
+	Reverse bool
+	Cursor  []byte
+	Limit   uint64
+}
+
+type DBGetIteratorArgs struct {
+	Prefix  string
+	Options GetIteratorOptions
+}
+
+type DBGetIteratorResult struct {
+	Limit uint64
```

Comment:
how about change field name from Limit to Count?
And... is it necessary?

Reply:
@kfangw I want to make `DBGetIteratorResult` to be similar with `ListOption` and I think it is necessary because the jsonrpc server don't want to serve huge data at once.

Reply:
It is the result, not input argument I guess.
Am I wrong?

Reply:
@kfangw Ah, sorry. I misunderstand. Anyway Good question 👍 I waited this question.

The `DBGetIteratorResult.Limit` is for notify the limit of server to the client,

1. if server limit is `100` and has `1000` data
1. client requests `200`
1. server send `100` result

This situation, client get the `100`, but he requests `200`. Client think no more result in this request, but server does not. Client will use `DBGetIteratorResult.Limit` server's limit.

Reply:
I see. It makes sense. Then, it should be named `Limit` as is.

Reply:
We already use limit in other places (the API, most prominently)